use-cases/using-vpp-and-kata.md

```diff
@@ -0,0 +1,76 @@
+# Setup to run VPP
+
+The Data Plane Development Kit (DPDK) is a set of libraries and drivers for
+fast packet processing. The Vector Packet Processing (VPP) is a platform
+extensible framework which provides out-of-the-box production quality
+switch and router functionality. VPP is a high performance packet-processing
+stack which can run on commodity CPUs. Enabling VPP with DPDK support can
+yield significant performance improvements over a Linux\*-bridge providing a
+switch with DPDK vhost-user ports.
+
+For more information about VPP visit their [wiki](https://wiki.fd.io/view/VPP).
+
+## Install and configure Kata Containers
+
+Follow  the [Kata Containers' setup instructions](https://github.com/kata-containers/documentation/wiki/Developer-Guide).
+
+In order to make use of vhost-user based interfaces, the container needs to be backed
+by huge pages.  After enabling huge-pages support on the host system, update the Kata configuration to
```

Comment:
I think it would be clearer to state somewhere in this paragraph that "huge pages" are a kernel feature.

Reply:
I think it will be useful to specify the kernel configuration that needs to be checked for huge pages as well. Along with the kernel configuration, it is also necessary to enable huge pages on the kernel command line as well iirc.
Will be useful to mention a check for that.